rust/parse_ast/src/convert_ast/converter.rs

```diff
@@ -556,10 +556,10 @@ impl<'a> AstConverter<'a> {
       ClassMember::PrivateMethod(private_method) => self.convert_private_method(private_method),
       ClassMember::PrivateProp(private_property) => self.convert_private_property(private_property),
       ClassMember::StaticBlock(static_block) => self.convert_static_block(static_block),
+      ClassMember::Empty(empty_stmt) => self.convert_empty_statement(empty_stmt),
```

Comment:
I think it is not correct to convert it to an empty statement. ESTree spec does not allow statements in classes https://github.com/estree/estree/blob/master/es2022.md#classbody
If you compare with what acorn does https://astexplorer.net you see that it just ignores the semicolon and does not generated an AST Node for it. I think we should do the same.

Reply:
Ah, I see.